unlock-js/src/erc20.js

```diff
@@ -32,8 +38,14 @@ export async function getErc20Decimals(erc20ContractAddress, provider) {
  */
 export async function getErc20TokenSymbol(erc20ContractAddress, provider) {
   const contract = new ethers.Contract(erc20ContractAddress, erc20abi, provider)
-  const symbolPromise = await contract.symbol()
-  return symbolPromise
+  let symbol
+  try {
+    symbol = await contract.symbol()
+  } catch (e) {
+    /** Some ERC20 contracts, including DAI do not have the right symbol method. */
+    return null
```

Comment:
What will we show on the paywall in this case?

Reply:
Should we do a hardcode for the DAI address as a stopgap?

Reply:
On the paywall, DAI is hardcoded for its adress, and others will show ERC20.
We can probably move that logic here at some point.